packages/e2e-tests/test-applications/standard-frontend-react/test-recipe.json

```diff
@@ -7,5 +7,13 @@
       "testName": "Playwright tests",
       "testCommand": "yarn test"
     }
+  ],
+  "canaryVersions": [
```

Comment:
Thought about testing previous versions of react here but it was a PITA to get working with react router so I just dropped it

Reply:
> react router

![shaking-fist-dark-mode](https://user-images.githubusercontent.com/18689448/195611819-7906df11-a5f7-433d-baaa-015740b24f4e.gif)

React is super backwards compatible so I wouldn't worry too much about it.